From 15f506042cf771d756046096841ee53d1cba7059 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Tue, 20 Jun 2017 14:28:50 -0400
Subject: [PATCH 1/7] Revert "Bluetooth: btmrvl_sdio: fix firmware activation
 failure"

This reverts commit 9a01242dc7fc4d5fe3f722afbf35b33aa414cd2f.

Boot times exceed 2 minutes on the Overo with this change.
---
 drivers/bluetooth/btmrvl_sdio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/bluetooth/btmrvl_sdio.c b/drivers/bluetooth/btmrvl_sdio.c
index d02f2c1..ba7fe04 100644
--- a/drivers/bluetooth/btmrvl_sdio.c
+++ b/drivers/bluetooth/btmrvl_sdio.c
@@ -433,7 +433,7 @@ static int btmrvl_sdio_verify_fw_download(struct btmrvl_sdio_card *card,
 		if (firmwarestat == FIRMWARE_READY)
 			return 0;
 
-		msleep(100);
+		msleep(10);
 	}
 
 	return -ETIMEDOUT;
-- 
2.7.4

